Don't crash when no database connection is present

arthurWD opened the pull request:
Fixes #607 and seems to be related to #589 and #608.

brett-anderson commented:
@ArthurWD Yes, this is a cleaner way to handle what I was trying to do. Should I close my PR #608 ?

rpbaltazar committed:
[Resolves] Basic support for Rails 6

- N/a

# Describe the changes made. What does this PR changes that might be critical. If any critical decisions have been made, make sure you explain the rationale for these decisions.

- Updated JRuby minor version to fix some tests
- Updated travis.yml to latest syntax
- Fixed Rakefile to run in rails 6
- Updated gemspec to have URLs matching

# How does the implementation addresses the problem

After merging this all test suite for apartment gem is passing using rails 6.

rpbaltazar committed:
[Hotfix] Updated readme

rpbaltazar committed:
[Hotfix] preparing version 2.3.0

rpbaltazar committed:
[Hotfix] Updated travis config and history.md

rpbaltazar committed:
[Hotfix] updated README to include require

rpbaltazar committed:
[Hotfix] Added story_branch local config

rpbaltazar committed:
[Hotfix] added rubocop to the project

rpbaltazar committed:
WIP - starting to cleanup

rpbaltazar committed:
WIP - ran autocorrect

rpbaltazar committed:
[Hotfix] Fixed modifier indentation

rpbaltazar committed:
WIP - fixed braces around hash params

rpbaltazar committed:
WIP - fixed array of strings

rpbaltazar committed:
[Hotfix] fixed array of strings for schemas rb

rpbaltazar committed:
WIP - removed a few more broken rules in rubocop

rpbaltazar committed:
WIP - cleanup a few more rubocop

rpbaltazar committed:
WIP - fixed a couple more of rubocops

rpbaltazar committed:
WIP -

rpbaltazar committed:
WIP - removed extra blank lines

rpbaltazar committed:
WIP - fixed EmptyLinesAroundBlockBody

rpbaltazar committed:
WIP - Fixed EmptyLinesAroundModuleBody

rpbaltazar committed:
WIP - ran more auto correct

rpbaltazar committed:
WIP - fixed SpaceBeforeBlockBraces

rpbaltazar committed:
WIP - fixed spacing related issues

rpbaltazar committed:
WIP - ran some more auto correct

rpbaltazar committed:
WIP - fixed dynamic find by

rpbaltazar committed:
WIP - fixed rails output and preferred alias method

rpbaltazar committed:
WIP - sorted gemfiels

rpbaltazar committed:
WIP - fixed unused argument

rpbaltazar committed:
WIP - fixed yaml safe load

rpbaltazar committed:
WIP - fixed block delimiters

rpbaltazar committed:
WIP - fixed keyword todo and empty method definition

rpbaltazar committed:
WIP - fixed expand path

rpbaltazar committed:
WIP - fixed a couple more rules

rpbaltazar committed:
WIP -

rpbaltazar committed:
WIP - fixed if unless modifier

rpbaltazar committed:
WIP - fixed multiline modifier

rpbaltazar committed:
WIP - fixed mutable constants

rpbaltazar committed:
WIP - fixed all immediatly fixable issues

rpbaltazar committed:
WIP - moved rubocop to gemfile instead of dev dependency

rpbaltazar committed:
WIP - moved rails 5.2 with ruby 2.2.9 to allowed failures

rpbaltazar committed:
WIP - added rubocop lint

rpbaltazar committed:
Merge pull request #10 from rails-on-services/fix-rubocop-usage

- Fixes CI runs:
  - One of the rails versions [had a bug](https://github.com/rails/rails/issues/38137) and causes the CI to fail. I've moved it to the allowed failures.
  - I think we should deprecate these versions as they are EOL. => New ticket to be created
- Fixes rubocop installation - I had not realized when I merged rubocop installation that I was breaking all CI (I'm sorry for the rushed decision). This fixes the dependencies and all CI should be passing.
- Adds rubocop to github actions. All the rubocop failures will be fixed with #8

rpbaltazar committed:
WIP - added base application record class

rpbaltazar committed:
WIP - fixed ignored_first_subdomains

rpbaltazar committed:
WIP - fixed failing specs after rubocop cleanup

rpbaltazar committed:
WIP - reverted sqlite3 as it is not an issue atm

rpbaltazar committed:
WIP - cleaning up rubocop

rpbaltazar committed:
WIP -

rpbaltazar committed:
Merge pull request #8 from rails-on-services/fix-rubocop

[Resolves #9] Cleanup rubocop todo

rpbaltazar committed:
WIP - working on travis config

rpbaltazar committed:
WIP - including master in branches to test

rpbaltazar committed:
Merge pull request #13 from rails-on-services/update-travis-config-to-only-run-pr-s-in-12

[Resolves #12] Update travis config to only run PRs instead of all commits

- Updated travis.yml to run only for branches: `master` and `development`.
- Due to the nature of PRs, nothing needs to be done to ensure tests run against an open PR

Commit:
Don't crash when no database connection is present

victorhazbun commented on lib/apartment/railtie.rb:
```diff
@@ -33,10 +34,15 @@ class Railtie < Rails::Railtie
         end
-      rescue ::ActiveRecord::NoDatabaseError
+      # rubocop:disable Lint/SuppressedException
+      rescue ::ActiveRecord::NoDatabaseError, PG::ConnectionBad
```
This is supposed to be the fix, correct?
